packages/markdown-docx/src/CiceroMarkToOOXML/index.js

```diff
@@ -138,6 +148,7 @@ class CiceroMarkToOOXMLTransfomer {
             this.getNodes(node, counter);
         });
         this.globalOOXML = wrapAroundDefaultDocxTags(this.globalOOXML);
+        this.counter = counter;
```

Comment:
Since `counter` is an instance variable now, you don't need to maintain a separate `counter`. Use `this.counter` wherever required.